src/output.cpp

```diff
@@ -223,34 +189,12 @@ static void HandleErrorOutput(const std::string& err) {
 }
 
 void Output::Quit() {
-	if (LOG_FILE) {
-		LOG_FILE.Close();
-	}
+	Game_Config::CloseLogFile();
 
-	int log_size = 1024 * 100;
-
-	char* buf = new char[log_size];
-
-	auto in = FileFinder::Save().OpenInputStream(OUTPUT_FILENAME, std::ios_base::in);
-	if (in) {
-		in.seekg(0, std::ios_base::end);
-		if (in.tellg() > log_size) {
-			in.seekg(-log_size, std::ios_base::end);
-			// skip current incomplete line
-			in.getline(buf, 1024 * 100);
-			in.read(buf, 1024 * 100);
-			size_t read = in.gcount();
-			in.Close();
-
-			auto out = FileFinder::Save().OpenOutputStream(OUTPUT_FILENAME, std::ios_base::out);
-			if (out) {
-				out.write(buf, read);
-			}
-		}
+	if (!Game_Config::GetLogFileOutput()) {
+		return;
```

Comment:
This makes no sense to me. I guess the closing brace is wrongly placed.

Reply:
right. This is nonsense (and already handled in ``Game_Config::CloseLogFile``